### ISSUE TITLE
Starting with empty DB, syncdb fails

### ISSUE THREAD

**Issue by vincic** [diefenbach/lfs-buildout-development#5]
Trying with new DB, syncdb fails.

```
  File "/Users/sasha/dev/github/lfs-buildout-development/eggs/Django-1.4.10-py2.7.egg/django/utils/importlib.py", line 35, in import_module                                                                                       
    __import__(name)                                                                                                                                                                                                              
  File "/Users/sasha/dev/github/lfs-buildout-development/src/lfs/lfs/manage/urls.py", line 54, in <module>                                                                                                                        
    urlpatterns += SEOView.get_seo_urlpattern(Manufacturer)                                                                                                                                                                       
  File "/Users/sasha/dev/github/lfs-buildout-development/src/lfs/lfs/manage/seo/views.py", line 30, in get_seo_urlpattern                                                                                                         
    ct = ContentType.objects.get_for_model(model_klass).pk                                                                                                                                                                        
  File "/Users/sasha/dev/github/lfs-buildout-development/eggs/Django-1.4.10-py2.7.egg/django/contrib/contenttypes/models.py", line 42, in get_for_model                                                                           
    defaults = {'name': smart_unicode(opts.verbose_name_raw)},                                                                                                                                                                    
  File "/Users/sasha/dev/github/lfs-buildout-development/eggs/Django-1.4.10-py2.7.egg/django/db/models/manager.py", line 134, in get_or_create                                                                                    
    return self.get_query_set().get_or_create(**kwargs)                                                                                                                                                                           
  File "/Users/sasha/dev/github/lfs-buildout-development/eggs/Django-1.4.10-py2.7.egg/django/db/models/query.py", line 445, in get_or_create                                                                                      
    return self.get(**lookup), False                                                                                                                                                                                              
  File "/Users/sasha/dev/github/lfs-buildout-development/eggs/Django-1.4.10-py2.7.egg/django/db/models/query.py", line 361, in get                                                                                                
    num = len(clone)                                                                                                                                                                                                              
  File "/Users/sasha/dev/github/lfs-buildout-development/eggs/Django-1.4.10-py2.7.egg/django/db/models/query.py", line 85, in __len__                                                                                             
    self._result_cache = list(self.iterator())                                                                                                                                                                                    
  File "/Users/sasha/dev/github/lfs-buildout-development/eggs/Django-1.4.10-py2.7.egg/django/db/models/query.py", line 291, in iterator                                                                                           
    for row in compiler.results_iter():                                                                                                                                                                                           
  File "/Users/sasha/dev/github/lfs-buildout-development/eggs/Django-1.4.10-py2.7.egg/django/db/models/sql/compiler.py", line 763, in results_iter                                                                                
    for rows in self.execute_sql(MULTI):                                                                                                                                                                                          
  File "/Users/sasha/dev/github/lfs-buildout-development/eggs/Django-1.4.10-py2.7.egg/django/db/models/sql/compiler.py", line 818, in execute_sql                                                                                 
    cursor.execute(sql, params)                                                                                                                                                                                                   
  File "/Users/sasha/dev/github/lfs-buildout-development/eggs/Django-1.4.10-py2.7.egg/django/db/backends/util.py", line 40, in execute                                                                                            
    return self.cursor.execute(sql, params)                                                                                                                                                                                       
  File "/Users/sasha/dev/github/lfs-buildout-development/eggs/Django-1.4.10-py2.7.egg/django/db/backends/sqlite3/base.py", line 344, in execute                                                                                   
    return Database.Cursor.execute(self, query, params)                                                                                                                                                                           
django.db.utils.DatabaseError: no such table: django_content_type
```

**Comment by vincic** [comment]
workaround is to set DEBUG=False in settings.py when running syncdb

**Comment by pigletto** [comment]
Please update your lfs app (src/lfs), it should be fixed now with: https://github.com/diefenbach/django-lfs/commit/fccb20f1d5b02bb8806b5cfc51e6f3b0669501e8

I've also updated django to 1.4.10 in buildout.cfg

**Comment by diefenbach** [comment]
Am 05.01.2014 um 17:41 schrieb Maciej Wiśniowski notifications@github.com:

> Please update your lfs app (src/lfs), it should be fixed now with: diefenbach/django-lfs@fccb20f
> 
> I've also updated django to 1.4.10 in buildout.cfg

Thanks Maciej!

Kai
